src/chart/components/high_low/high-low.drawer.ts

```diff
@@ -9,6 +9,7 @@ import { FullChartConfig } from '../../chart.config';
 import { ChartModel } from '../chart/chart.model';
 import { CanvasModel } from '../../model/canvas.model';
 import { getTextLineHeight } from '../../utils/canvas/canvas-text-functions.utils';
+import { clipToBounds } from '../../drawers/data-series.drawer';
```

Comment:
pls, move the `clipToBounds` function to `utils/canvas/canvas-drawing-functions.utils.ts` and update all the imports